modules/rosa-sts-interactive-cluster-creation-mode-options.adoc

```diff
@@ -26,7 +26,10 @@ The following table describes the interactive cluster creation mode options:
 |Enable the use of Hosted Control Planes.
 
 |`Create cluster admin user`
-|Create a cluster administrator user when you create your cluster using the htpasswd identity provider. The username must not contain `/`, `:`, or `%`. The password must be at least 14 characters (ASCII-standard) without whitespaces.
+a|Create a local administrator user (`cluster-admin`) for your cluster. This automatically configures a htpasswd identity provider for the `cluster-admin` user.
+
+|`Create custom password for cluster admin`
+a|Create a custom password for the `cluster-admin` user, or use a system-generated password. If you create a custom password, the password must be at least 14 characters (ASCII-standard) and contain no whitespace characters. If you do not create a custom password, the system generates a password and displays it the command line output.
```

Comment:
`the system generates a password and displays it the command line output` 
i suggest to replace above with
`the system generates a password and displays it on the command line output`

Reply:
Thanks for the catch!